impl/src/main/java/org/jboss/forge/roaster/model/impl/JavaPackageInfoImpl.java

```diff
@@ -117,33 +92,20 @@ public PackageDeclaration getPkg()
    }
 
    @Override
-   public JavaDocSource<JavaPackageInfoSource> getJavaDoc()
-   {
-      Javadoc javadoc = pkg.getJavadoc();
-      if (javadoc == null)
-      {
-         javadoc = pkg.getAST().newJavadoc();
-         pkg.setJavadoc(javadoc);
-      }
-      return new JavaDocImpl<>(this, javadoc);
-   }
-
-   @Override
-   public JavaPackageInfoSource removeJavaDoc()
+   protected JavaPackageInfoSource updateTypeNames(String name)
    {
-      pkg.setJavadoc(null);
-      return this;
+      throw new ParserException();
```

Comment:
throw new UnsupportedOperationException()?

Reply:
Afaik this method doesn't seem to be used anywhere or am I missing something?

Reply:
Let me check, but I needed to overwrite it

Reply:
I need to overwrite it, since the it's abstract by definition. The method needs to be abstract , since it's required in JavaSourceImpl but cannot have a body, since no data is in this class to work on.
If you like, I can just overwrite the method and do nothing, but I like it more to throw a exception since this method shoudn't be called inside a package-info. In the case it's called by a bug, we'll see it directly